messaging/rabbitmq_binding.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+
```

Comment:
Should I had an header like the other modules that state it's part of ansible-modules or a generic one like MIT?